### PR TITLE
Drop duplicate version statement in README.md

### PR DESCRIPTION
**Pull request by proinsias**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ The original SAS implementation can be found [here](https://www.nber.org/data/cm
 Currently, hccpy supports CMS-HCC V22, V23, V24, ESRD, and HHS-HCC Year 2019.
```
Comment (hooley-pomelo) [inline README.md:7]: It also covers HHS-HCC Year 2021 (needs to be added)

Comment (proinsias) [reply README.md]: I've added this and tidied up the text a bit. Let me know what you think.

**Review by dr00b — CHANGES_REQUESTED** [review]
Thank you @proinsias for cleaning up the README. I believe HHS-HCC 2021 is incorrect however, I believe it's 2022. Can you change, then I will approve and merge!

**Review by dr00b — APPROVED** [review]
Thanks again!

**Pull request merged**